integrations/dynatrace/main.py

```diff
@@ -0,0 +1,58 @@
+from enum import StrEnum
+from typing import Any
+
+from loguru import logger
+from port_ocean.context.ocean import ocean
+
+from client import DynatraceClient
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+
+class ObjectKind(StrEnum):
+    PROBLEM = "problem"
+    SLO = "slo"
+    ENTITY_TYPE = "entity_type"
+    ENTITY = "entity"
+
+
+def initialize_client() -> DynatraceClient:
+    return DynatraceClient(
+        ocean.integration_config["dynatrace_host"],
+        ocean.integration_config["dynatrace_api_key"],
```

Comment:
In the spec.yaml and config.yaml, we have `dynatraceApiKey`, `dynatraceHostUrl`

the integration_config uses the same variables but in underscore. this means you need to change these to

`dynatrace_host_url` and `dynatrace_api_key` (which you got right here)